access/passwords.md

```diff
@@ -10,12 +10,12 @@ This policy applies to all Lullabot employees and contractors.
 All employees and contractors are expected to create strong passwords for access to all Lullabot and client accounts.
 
 ### Explanation and Implementation
-The first layer of defense that we have for our online accounts is the PIN, passcode, or password. As such, it is extremely important to use good, unique passwords, and keep them well protected. A good password consists of a fully random string, the longer the better. Contrary to popular belief, the inclusion of numbers, characters, or mixed case does not matter nearly as much as the length of the password itself.
+The first layer of defense that we have for our online accounts is the PIN, passcode, or password. As such, it is extremely important to use good, unique passwords, and keep them well protected. A good password consists is hard for people to guess and machines to brute-force. Contrary to popular belief, the inclusion of numbers, characters, or mixed case does not matter nearly as much as the length of the password itself.
```

Comment:
```suggestion
The first layer of defense that we have for our online accounts is the PIN, passcode, or password. As such, it is extremely important to use good, unique passwords, and keep them well protected. A good password is difficult for people to guess and machines to brute-force. Contrary to popular belief, the inclusion of numbers, characters, or mixed case does not matter nearly as much as the length of the password itself.
```

Reply:
"consists is" -> "is"
"hard" -> "difficult" ... personal preference. difficult is more accurate, and translates to similar words in other languages.